makerdao/governance-portal-v2
# Chore/sc 519/upgrade swr to 1 0

### Link to Clubhouse story

https://app.shortcut.com/dux-makerdao/story/519/upgrade-swr-to-1-0

### What does this PR do?

Upgrades to SWR 1.0

Reduces default polling interval

Cuts down on unnecessary calls

Adds mutations (revalidation) where necessary

### Steps for testing:

Check all the pages of the app. Send tests transactions. Look for anything funky.

### Any additional helpful information?:

Reducing infura calls will be handled in a separate PR

### Add a GIF:
![](https://media.giphy.com/media/RMwcpN4Q3ZN4A4TaLL/giphy.gif)
